config/webpacker.yml

```diff
@@ -31,7 +31,7 @@ default: &default
 
 development:
   <<: *default
-  compile: true
+  compile: false
```

Comment:
If you change this, the rest of the team that is not working actively with JS, will need to trigger manually the build.